File: Dia35/NumerosAleatorios2.js
// Números Aleatorios 2 en JavaScript

function randomNumbers2(dices, faces) {
    let results = new Array()
    for (let d = 0; d < dices; d++) {
        results.push(Math.ceil(Math.random() * faces))
    }
    return results
}

const numeroDados = parseInt(prompt('¿Cuántos dados vas a tirar?'))
const numeroCaras = parseInt(prompt('¿Cuántos caras tienen?'))
alert(`Sacaste: ${randomNumbers2(numeroDados, numeroCaras)}`)

File: Dia35/NumerosAleatorios2.py
# Números Aleatorios 2 en Python

from random import choice

def randomNumbers2(dices, faces):
    results = []
    for d in range(0, dices):
        results.append(choice(range(1, faces + 1)))
    return results

numeroDados = int(input('¿Cuántos dados vas a tirar?: '))
numeroCaras = int(input('¿Cuántos caras tienen?: '))
print('Sacaste: {}'.format(randomNumbers2(numeroDados, numeroCaras)))